opentelemetry-sdk/src/trace/span_processor.rs

```diff
@@ -92,6 +93,8 @@ pub trait SpanProcessor: Send + Sync + std::fmt::Debug {
     /// Shuts down the processor. Called when SDK is shut down. This is an
     /// opportunity for processors to do any cleanup required.
     fn shutdown(&mut self) -> TraceResult<()>;
+    /// Set the resource for the log processor.
+    fn set_resource(&mut self, _resource: &Resource);
```

Comment:
provide an empty {}, so we can go easy on those test processors who don't need this

Reply:
Yes, done.